### ISSUE TITLE
many-body-contact-comparison isn't GPU friendly

### ISSUE THREAD

**Issue by evanberkowitz** [evanberkowitz/two-dimensional-gasses#90]
A number of issues
 - [ ] ere.analytic didn't like getting CPU data.
 - [ ] Many `tdg.references`' plotting routines assume the data is on the CPU.
 - [ ] A variety of `plt.{plot,errorbar}` can't take GPU data.

We should probably provide some syntactic sugar for (some well-considered sequence of) `.clone().detach().cpu().numpy()` or something...
